crates/bgp-pkt/src/path_attribute/path_attribute.rs

```diff
@@ -945,9 +945,9 @@ impl ClusterId {
 ///
 /// 1. to advertise a feasible route to a peer
 /// 2. to permit a router to advertise the Network Layer address of the router
-/// that should be used as the next hop to the destinations
-/// listed in the Network Layer Reachability Information field of the `MP_NLRI`
-/// attribute.
+///     that should be used as the next hop to the destinations
+///     listed in the Network Layer Reachability Information field of the `MP_NLRI`
+///     attribute.
```

Comment:
usuaully not good idea to include non-related items to the PR

Reply:
sadly could not get the CI to pass without those changes

Reply:
maybe could you fix this on main then i can rebase and these changes will not appear anymore ?

Reply:
Good catch, seem the new nightly version of rust just got strict about these rules. Don't worry, I just updated main.

Reply:
great thanks :)